src/main/java/uk/gov/hmcts/reform/bulkscanccdeventhandler/transformation/services/DocumentMapper.java

```diff
@@ -14,9 +14,11 @@
      */
     public Item<ScannedDocument> toCaseDoc(
         InputScannedDoc exceptionRecordDoc,
-        String exceptionRecordReference
+        String exceptionRecordReference,
+        boolean isAutomatedProcess
     ) {
-        if (exceptionRecordDoc == null) {
+        if (!isAutomatedProcess && exceptionRecordDoc == null) {
```

Comment:
Currently, when the exception record id is null, transformation result returns NO documents. 
But the Automated case creation request doesn't contain exception record id so transformation service should return scanned documents with no exception record id (which is done in the else part)